static/app/views/organizationStats/index.spec.tsx

```diff
@@ -179,6 +186,7 @@ describe('OrganizationStats', function () {
 
     expect(await screen.findByTestId('usage-stats-chart')).toBeInTheDocument();
     expect(screen.getByTestId('usage-stats-table')).toBeInTheDocument();
+    await waitForElementToBeRemoved(() => screen.queryByTestId('loading-indicator'));
     expect(screen.getByTestId('empty-message')).toBeInTheDocument();
```

Comment:
I tend to switch to a findBy instead of waiting for loading state to be removed because waitForElementToBeRemoved can flake if the loading state is not found.
```suggestion
    expect(await screen.findByTestId('empty-message')).toBeInTheDocument();
```